ci-operator/config/openshift/cluster-api-provider-openstack/openshift-cluster-api-provider-openstack-master.yaml

```diff
@@ -0,0 +1,45 @@
+base_images:
+  base:
+    cluster: https://api.ci.openshift.org
+    name: origin-v4.0
+    namespace: openshift
+    tag: base
+binary_build_commands: make build
+canonical_go_repository: sigs.k8s.io/cluster-api-provider-openstack
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  to: openstack-machine-controllers
+resources:
+  '*':
+    limits:
+      cpu: '2'
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  cluster: https://api.ci.openshift.org
+  name: origin-v4.0
+  namespace: openshift
+  tag: ''
+  tag_overrides: {}
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.10
+tests:
+- as: unit
+  commands: make test
```

Comment:
This is the odd man out, does it need `IS_CONTAINER=TRUE`?

Reply:
I copied this file from the AWS one. What is `IS_CONTAINER` used for?

Reply:
This field defines what commands get run for the 'unit' test as per the [doc](https://github.com/openshift/ci-operator/blob/master/CONFIGURATION.md#testscommands)

Reply:
Thanks for the link to the doc. These commands should be correct for this image too. I'll go through the doc and update the PR accordingly. Thanks again :cake: